conf/application.conf

```diff
@@ -1,4 +1,9 @@
 play.application.loader=AppLoader
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline'"
```

Comment:
This was necessary to allow inline styling, which we use to colour the rule pills. I'm happy to discuss if anyone's worried about the security implications.